docker-compose.yml

```diff
@@ -27,20 +27,6 @@ services:
       network:
     image: cypress/included:7.3.0
     working_dir: /app
-    depends_on:
-      - server
-    volumes:
-      - .:/app:delegated
-    entrypoint: npx cypress run-ct
-  cypress_update:
-    environment:
-      - CYPRESS_updateSnapshots=true
-    networks:
-      network:
-    image: cypress/included:7.3.0
-    working_dir: /app
-    depends_on:
-      - server
```

Comment:
I'm passing -e updateSnapshots=true in the command so I don't need a separate service just to set an environment var